addons/account_edi_ubl/models/account_edi_format.py

```diff
@@ -24,19 +34,7 @@ def _import_ubl(self, tree, invoice):
         :returns:       the invoice where the UBL data was imported.
         """
 
-        def _get_ubl_namespaces():
-            ''' If the namespace is declared with xmlns='...', the namespaces map contains the 'None' key that causes an
-            TypeError: empty namespace prefix is not supported in XPath
-            Then, we need to remap arbitrarily this key.
-
-            :param tree: An instance of etree.
-            :return: The namespaces map without 'None' key.
-            '''
-            namespaces = tree.nsmap
-            namespaces['inv'] = namespaces.pop(None)
```

Comment:
Are you sure this is not specific to UBL?

Reply:
See https://github.com/odoo/odoo/pull/68514#discussion_r604777776